forms/jqm/element.js

```diff
@@ -68,9 +71,13 @@ define(function (require) {
     renderLabel: function () {
       if (!this.$label) {
         this.$label = $('<label class="ui-input-text"></label>');
+      }
+
+      if (!this.$el.find(this.$label).length) {
```

Comment:
That `if (!this.$label) {` block should:

- create the element
- append it to `this.$el`
- save a reference to it as `this.$label`

So yeah, after the block there shouldn't be any reason to do a jQuery search for the element. Everywhere else in the View, we should just check whether `this.$label` is set and act accordingly.